Jenkinsfile.sdk

```diff
@@ -185,7 +185,11 @@ pipeline {
 								rm -rf coverage_elements; mkdir -p coverage_elements
 								for d in elements/*; do
 									if [ -d $d  ] && [ -d $d/.nyc_output  ]; then
-										cp -r $d/.nyc_output/* ./coverage_elements
+										if [$d = "lisk-p2p"]; then
+											cp -r $d/.coverage/* ./coverage_elements
+										else
+											cp -r $d/.nyc_output/* ./coverage_elements
+										fi
```

Comment:
Am I missing something or can the whole thing be replaced by a one-liner?
```
cp -f elements/*/.nyc_output/* elements/*/.coverage/* coverage_elements/
```

Reply:
ping @Nazgolze

Reply:
yeah, I think that's a better way to go